gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -336,7 +336,7 @@ func (vm *VMKeeper) AddPackage(ctx sdk.Context, msg MsgAddPackage) (err error) {
 	}
 	creatorAcc := vm.acck.GetAccount(ctx, creator)
 	if creatorAcc == nil {
-		return std.ErrUnknownAddress(fmt.Sprintf("account %s does not exist", creator))
+		return std.ErrUnknownAddress(fmt.Sprintf("account %s has not been initialized. does it have coins?", creator))
```

Comment:
it's a rhetorical question - accounts that don't exist don't have coins.

I'd prefer something like `account %s does not exist, it must receive coins to be created` so we point the user to a clearer problem and solution